internal/commands/user/disable_test.go

```diff
@@ -0,0 +1,324 @@
+package user
+
+import (
+	"errors"
+	"strings"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestDisableSetup(t *testing.T) {
```

Comment:
[nit] keep the test func naming pattern consistent here and use `TestUserDisable*`